src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -513,12 +441,10 @@ private bool ValidateNonce(PublicEntry entry, ExecutionEnvironment env, ITxTrace
             ITxTracer txTracer)
         {
             var senderBalance = _stateProvider.GetBalance(env.Sender);
-            if (intrinsicGas * env.GasPrice + env.Value <= senderBalance)
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"INSUFFICIENT_SENDER_BALANCE: ({env.Sender})_BALANCE = {senderBalance.ToString()}");
+            if (intrinsicGas * env.GasPrice + env.Value <= senderBalance) return true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)